common/gpu/amd/sea-islands/default.nix

```diff
@@ -0,0 +1,4 @@
+{
+  imports = [ ../. ];
+  boot.kernelParams = [ "radeon.cik_support=0" "amdgpu.cik_support=1" ];
```

Comment:
Can you explain what this module enables and why we only have it for certain generations?

Reply:
Certain AMD cards without these flags will refuse to use the amdgpu driver and only run with the Radeon driver, despite the amdgpu driver working absolutely fine and often better, I've had these parameters and the `si` equivilents in my common configuration for years now.

The Arch wiki provides a little more insight on this too: https://wiki.archlinux.org/index.php/AMDGPU

Reply:
Ok. It would be great if this would be commented in a comment above the flag.

Reply:
I wonder if we should still explicitly set the `radeon.cik_support=0` parameter when the radeon driver can be blacklisted by `nixpkgs/nixos/modules/hardware/video/amdgpu.nix`?

Reply:
Makes sense.

Reply:
Ok. From my side I only would add a comment here.